psh/rm/rm.c

```diff
@@ -0,0 +1,87 @@
+/*
+ * Phoenix-RTOS
+ *
+ * rm - unlink files or remove empty directories
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Gerard Swiderski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <sys/stat.h>
+
+#include "../psh.h"
+
+
+static void psh_rm_info(void)
+{
+	printf("unlink files or remove empty directories");
+}
+
+
+static void psh_rm_usage(void)
+{
+	printf("Usage: rm [-d] FILE...\n");
+}
+
+
+static int psh_rm(int argc, char **argv)
+{
+	struct stat stbuf;
+	int i, err, rdir = 0, ret = EXIT_SUCCESS;
+
+	if ((argc > 1) && (argv[1][0] == '-') && (argv[1][1] == 'd')) {
```

Comment:
```suggestion
	if ((argc > 1) && (argv[1][0] == '-') && (argv[1][1] == 'd') && (argv[1][2] == '\0')) {
```
prevents, for example, such calling:
```
psh_rm -def file
```

Reply:
Thanks. Done.